traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -104,7 +104,7 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		{1.1, http.MethodGet, `cdns/configs$`, handlerToFunc(proxyHandler), 0, NoAuth, []Middleware{}},
 		{1.1, http.MethodGet, `cdns/domains$`, handlerToFunc(proxyHandler), 0, NoAuth, []Middleware{}},
 		{1.1, http.MethodGet, `cdns/health$`, handlerToFunc(proxyHandler), 0, NoAuth, []Middleware{}},
-		{1.1, http.MethodGet, `cdns/routing$`, handlerToFunc(proxyHandler), 0, NoAuth, []Middleware{}},
```

Comment:
We need to solve custom TO extensions or the ability to override for custom project's needs

Reply:
yes, because TO_golang has no concept of to_extensions (at the moment), merging this PR  would kill the perl to_extension that is currently being used by Comcast in production.